### ISSUE TITLE
Cookie banner not blocked: studimup.de

### ISSUE THREAD

**Issue by bestoernesto** [ryanbr/fanboy-adblock#1049]
### List the website(s) you're having issues:
`https://www.studimup.de/`

### What happens?
Cookie banner not blocked

### List Subscriptions you're using:
fanboy's cookie list

### Your settings

- OS/version: Win7 (64bit)
- Browser/version: FF 56.0.2
- Adblock Extension/version: uBlock Origin 1.19.2

**Comment by ryanbr** [comment]
Already blocked? do you have another list using $generichide on the site?

**Comment by bestoernesto** [comment]
Hello, I'm afraid I don't know my way around. In which lists should I search for "$generichide"? 

However, I noticed the following. Yesterday I had opened many tabs (over 500) and worked with firefox for a long time (over 12 hours). After I reported "studimup.de"-cookie banner, more cookie banners were not blocked. And there were some banners that I knew for sure had been blocked a few hours before. But it was not the same for every website. On many pages the cookie banner was still blocked, but on approximately every 10th page a banner appeared again. By the way, the same situation arose with advertising banners. But after closing Firefox and restarting with all(!) previously opened tabs, the situation behaved as desired and planned. In the future I will close Firefox more often to avoid such problems.

(PS. With "DeepL" translated from German)
